Flutter/installation/Installation-errors.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Installation Errors in Flutter - Syncfusion
 description: Learn here about the common installation errors and solution to those errors in Syncfusion Flutter Components installation.
-platform: Flutter
+platform: flutter
```

Comment:
Hi @AswiniDileep,

Why the casing of the Flutter is changed here?

Regards,
Hari Hara Sudhan. K.